src/components/select/select.js

```diff
@@ -74,6 +74,8 @@ angular.module('material.components.select', [
  * @param {boolean=} multiple When present, allows for more than one option to be selected.
  *  The model is an array with the selected choices. **Note:** This attribute is only evaluated
  *  once; it is not watched.
+ * @param {string=} selection-separator-char Options available is limited to `comma`(Default),`semi-colon`, `pipe`.
+ * @param {string=} selection-separator-spacing Options available is limited to `after`(Default), `before`, `none`, `break`.
```

Comment:
Let's display both of these sets of options in a bulleted list like we do [here](https://material.angularjs.org/latest/api/directive/mdMenu#attributes) using [this code](https://github.com/cobuscoetsee/material/blob/8db610681bf06b610b6ade12129deb37564bac27/src/components/menu/js/menuDirective.js#L172-L185).